File: go_services/find_service_test.go
package go_services_test

import (
	. "github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
	"github.com/sporto/service_objects/go_services"
)

var _ = Describe("FindService", func() {
	var (
		serv *go_services.FindService
	)

	BeforeEach(func() {
		serv = &go_services.FindService{}
	})

	It("returns a slice", func() {
		Expect(serv.Run()).To(Equal([]int{1,2}))
	})
})

File: js/find_service_test.js
// var assert = require("assert");
var Service = require('./find_service');
var expect = require('chai').expect;

describe('FindService', function(){

	var serv;

	beforeEach(function () {
		serv = Service();
	});

	describe('.run', function(){
		it('returns an array', function(){
			var res = serv.run();
			expect(res).to.eql([1, 2]);
		})
	});
})